lib/WebpackOptionsApply.js

```diff
@@ -72,12 +73,19 @@ WebpackOptionsApply.prototype.process = function(options, compiler) {
 			case "web":
 				JsonpTemplatePlugin = require("./JsonpTemplatePlugin");
 				NodeSourcePlugin = require("./node/NodeSourcePlugin");
+
 				compiler.apply(
 					new JsonpTemplatePlugin(options.output),
 					new FunctionModulePlugin(options.output),
 					new NodeSourcePlugin(options.node),
 					new LoaderTargetPlugin("web")
 				);
+
+				if (options.performance.hints) {
+					EmittedAssetSizeLimitPlugin = require("./performance/EmittedAssetSizeLimitPlugin");
+					compiler.apply(new EmittedAssetSizeLimitPlugin(options.performance));
+				}
```

Comment:
:+1: I think it makes sense to use this plugin for other targets too. I. e. webworker, etc.

Could you move it out of the target section and apply it always?
In addition to this change the `hint` default to `false` except if `target = web`.